flutter_kernel_transformers/lib/track_widget_constructor_locations.dart

```diff
@@ -439,6 +455,14 @@ class WidgetCreatorTracker {
       return;
     }
 
+    // TODO(jacobr): once there is a working incremental ClassHierarchy
+    // constructor switch to using it instead of building a ClassHierarchy off
+    // the full program.
+    hierarchy = new ClassHierarchy(
```

Comment:
Given that this constructor of `ClassHierarchy` is now used, calling `hierarchy.applyChanges(transformedClasses)` below (line 484) doesn't really do anything.  I would suggest rebuilding the class hierarchy entirely in the case of updates, if that's really needed.

Reply:
Good point. removed all uses of applyChanges.